<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-L54GKRA" failures="0" tests="4" name="testscripts.Logintest" time="68.233" errors="1" timestamp="2025-05-26T19:24:00 IST" skipped="0">
  <testcase classname="testscripts.Logintest" name="invalidpasswordcrctusername" time="18.997"/>
  <testcase classname="testscripts.Logintest" name="verifyuserloginwithvalidcreditions" time="10.418">
    <error message="C:\Users\User\git\grocery-application\Mainproject1C:\Users\User\eclipse-workspace\Mainproject1\src\test\resources\7rmart\GroceryApp.xlsx (The filename, directory name, or volume label syntax is incorrect)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\Users\User\git\grocery-application\Mainproject1C:\Users\User\eclipse-workspace\Mainproject1\src\test\resources\7rmart\GroceryApp.xlsx (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at utilities.Excelread.readStringData(Excelread.java:20)
at page.Loginpage.loginusingexcel(Loginpage.java:51)
at testscripts.Logintest.verifyuserloginwithvalidcreditions(Logintest.java:22)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </error>
  </testcase> <!-- verifyuserloginwithvalidcreditions -->
  <testcase classname="testscripts.Logintest" name="validincrctpassincrtusername" time="24.553"/>
  <testcase classname="testscripts.Logintest" name="validpassincrtusername" time="14.265"/>
</testsuite> <!-- testscripts.Logintest -->
